pkg/filesystem/fs/fs.go

```diff
@@ -554,6 +555,12 @@ func (fs *Filesystem) Mount(snapshotID string, labels map[string]string) (err er
 	return nil
 }
 
+// How much space the layer/blob cache filesystem is occupying
+func (fs *Filesystem) CacheUsage(ctx context.Context, snapshotID, blobDigest string) (snapshots.Usage, error) {
```

Comment:
Seems `snapshotID` is a useless parameter.

Reply:
fixed